TCAT/Controllers/CustomNavigationController.swift

```diff
@@ -99,6 +132,21 @@ class CustomNavigationController: UINavigationController, UINavigationController
     @objc private func backAction() {
         _ = popViewController(animated: true)
     }
+    
+    @objc func reachabilityChanged(notif: Notification) {
+        if let reachability = notif.object as? Reachability {
+            switch reachability.connection {
+            case .wifi, .cellular:
+                banner.dismiss()
+            case .none:
+                banner.show(queuePosition: .front, on: self)
+                banner.autoDismiss = false
+                banner.isUserInteractionEnabled = false
```

Comment:
Ik this was from previous code but can we take this line out? I don't think we should make this banner not dismissible.

Reply:
It makes sense to me to not make the banner dismissible by the user though. I feel we don't want the user to be able to dismiss the banner bc they might get confused if they dismissed it and can't distinguish between when they have internet or not.